docs/src/assets/german_credit.csv

```diff
@@ -0,0 +1,1001 @@
+,Age,Sex,Job,Housing,Saving accounts,Checking account,Credit amount,Duration,Purpose

```

Comment:
why rowid column does not have a name?

Reply:
Oh, yeah. Fixing it. In the kaggle they had mentioned using #.